resources/session02/homework/http_server.py

```diff
@@ -32,10 +35,25 @@ def parse_request(request):
         raise NotImplementedError("We only accept GET")
     return uri
 
-
 def resolve_uri(uri):
     """This method should return appropriate content and a mime type"""
-    return b"still broken", b"text/plain"
+    homedir = str(pathlib.Path.cwd().joinpath('webroot'))
+    fullpath = pathlib.Path(homedir + uri)
```

Comment:
What happens if URI is not absolute (i.e `images/image.jpeg` instead of `/images/image.jpeg`)?